inspirehep/modules/records/serializers/templates/latex_template.tex

```diff
@@ -0,0 +1,54 @@
+%\cite{\VAR{ data.texkeys }}
```

Comment:
The problem is, whitespace control doesn't only control end of lines, but also spaces, meaning that if we want a new string to be printed with a whitespace at the beginning but no end of line, you have to put the whole ``if`` to ``endif`` in the same line. If you do:
```
first text
{%- if condition -%}
 some more text
{%- endif %}
```
(which I understand would be the correct way to write it AND use the whitespaces), if the condition is met, the result would be: ``first textsome more text``
By letting it print the whitespaces, it will print the initial space before ``some other text`` but also the start of a new line. By putting everything in the same line, there is no new line to be printed.